shiftleft-python-demo/flask_webgoat/__init__.py

```
@@ -0,0 +1,34 @@
+
+def create_app():
+    app = Flask(__name__)
```

Ruff (0.15.6)

[error] Undefined name `Flask`

(F821)